src/components/MyAccount/CheckoutsTab.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  Box,
+  Icon,
+  Link,
+  Table,
+  Text,
+} from "@nypl/design-system-react-components"
+import type { Checkout, Patron } from "../../types/accountTypes"
+import styles from "../../../styles/components/MyAccount.module.scss"
+import RenewButton from "./RenewButton"
+
+const CheckoutsTab = ({
+  checkouts,
+  patron,
+}: {
+  checkouts: Checkout[]
+  patron: Patron
+}) => {
+  const checkoutsHeaders = [
+    "Title",
+    "Barcode",
+    "Call number",
+    "Due back by",
+    "Manage checkout",
+  ]
+
+  function formatTitle(checkout: Checkout) {
+    if (checkout.href) {
+      return <Link href={checkout.href}>{checkout.title}</Link>
+    } else {
+      return <Text>{checkout.title}</Text>
+    }
+  }
+
+  const checkoutsData = checkouts.map((checkout) => [
+    formatTitle(checkout),
+    checkout.barcode,
+    checkout.callNumber,
+    checkout.dueDate,
+    checkout.isResearch ? null : RenewButton(checkout, patron),
```

Comment:
Can the RenewButton be used more like a component? as in, turning up here as `<RenewButton checkout={checkout} patron={patron}/>`?